pkg/providers/pricing/zz_generated.pricing_aws.go

```diff
@@ -16,7 +16,7 @@ limitations under the License.
 
 package pricing
 
-// generated at 2023-12-11T13:07:33Z for us-east-1
```

Comment:
Note: we'll want to drop these generated pricing changes by the time it's ready to merge.